api/v1/models/user_model.py

```diff
@@ -0,0 +1,28 @@
+"""USER ENTITY """
+import uuid
+from werkzeug.security import generate_password_hash
+USERS = []
```

Comment:
@Busobozihakiim , we should make this(USER =[]) a class variable.
 read more about class variables, please. right now it is a global variable. we should try to ensure everything is OOP

Reply:
This has been implemeted